Updating Constraints Documentation (#282)
shannon-lab/zapdos: doc/content/source/constraints/ArbitrarilyTiedValueConstraint.md

# ArbitrarilyTiedValueConstraint

!syntax description /Constraints/ArbitrarilyTiedValueConstraint

## Overview

> Similar to [`TiedValueConstraint`](TiedValueConstraint.md), except the value of a variable is forced based on

Reviewer: ```suggestion
Similar to [TiedValueConstraint.md], except the value of a variable is forced based on
```

Let's use this shorthand to use the title of the linked page as the link text.

a user defined ratio at an interface.

> !alert warning title=Untested Class

Reviewer: ```suggestion
!alert! warning title=Untested Class
```

I like the use the full syntax for multi-line alerts.

The `ArbitrarilyTiedValueConstraint` does not have a formalized test, yet. For this reason,

> users should be aware of unforeseen bugs when using `ArbitrarilyTiedValueConstraint`. To

Reviewer: ```suggestion
users should beware of unforeseen bugs when using `ArbitrarilyTiedValueConstraint`. To
```

report a bug or discuss future contributions to Zapdos, please refer to the
[Zapdos GitHub Discussions page](https://github.com/shannon-lab/zapdos/discussions).
For standards of how to contribute to Zapdos and the MOOSE framework,

> please refer to the [MOOSE Contributing page](framework/contributing.md).

Reviewer: ```suggestion
please refer to the [MOOSE Contributing page](framework/contributing.md).
!alert-end!
```

## Example Input File Syntax

> !! Describe and include an example of how to use the ArbitrarilyTiedValueConstraint object.
> 

Reviewer: ```suggestion
```
